src/Tribe/Tickets_View.php

```diff
@@ -196,7 +197,13 @@ public function update_tickets() {
 
 		$post_id = get_the_ID();
 
-		$attendees = ! empty( $_POST['attendee'] ) ? $_POST['attendee'] : [];
+		$attendees = [];
+
+		if ( isset( $_POST['tribe_tickets']['attendees'] ) ) {
+			$attendees = $_POST['tribe_tickets']['attendees'];
+		} elseif ( isset( $_POST['attendee'] ) ) {
+			$attendees = $_POST['attendee'];
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_POST['attendee']
* :no_entry_sign: Processing form data without nonce verification.

<sub>hash: 6d5d55d30bb79df17317cc745bd9c51d</sub>